phs.ttl

```diff
@@ -113,24 +113,53 @@
 #################################################################
 
 ###  http://linked.data.gov.au/def/phs#taxonomicReferenceSource
-:taxonomicReferenceSource rdf:type owl:DatatypeProperty ;
-                          rdfs:domain [ rdf:type owl:Class ;
-                                        owl:unionOf ( :Host
-                                                      :SampleAnimal
-                                                    )
-                                      ] ;
-                          rdfs:comment "" ;
-                          skos:prefLabel "taxonomic reference source" .
-
-###  http://linked.data.gov.au/def/phs#vernacularName
-:vernacularName rdf:type owl:DatatypeProperty ;
-                rdfs:domain [ rdf:type owl:Class ;
-                              owl:unionOf ( :Host
-                                            :SampleAnimal
-                                          )
-                            ] ;
-                rdfs:comment "The vernacular (common) name for the host or pest" ;
-                skos:prefLabel "vernacular name" .
+:taxonomicRefSource rdf:type owl:DatatypeProperty ;
+        rdfs:domain [ rdf:type owl:Class ;
+                    owl:unionOf ( :Host
+                                  :Pest
+                                  :Bycatch
+                                )
+                  ] ;
+        rdfs:comment "Name of the taxonomic reference system that is the source of the taxonID" ;
+        rdfs:comment "Examples are 'GBIF', 'TaxaaS'." ;
+        rdfs:comment "This could be replaced or augmented with 'taxonomicRefSourceIRI'" ;
+        skos:prefLabel "Taxonomic reference source" .
+
+###  http://linked.data.gov.au/def/phs#taxonID
+:taxonID rdf:type owl:DatatypeProperty ;
```

Comment:
needs review. Might be bringing in too much complexity with union statement - but the use case needs to be stated.

@F3RR3T  had a simple goal of using the "common name" preflabel. So the use of this with taxonID may be incorrect